testing/e2e/app/routes/MiddleTextTruncation/spec.ts

```diff
@@ -0,0 +1,82 @@
+import { test, expect, Page } from '@playwright/test';
+
+test.describe('MiddleTextTruncation', () => {
+  const longItem =
+    'MyFileWithAReallyLongNameThatWillBeTruncatedBecauseItIsReallyThatLongSoHardToBelieve_FinalVersion_V2.html';
+
+  test(`should overflow whenever there is not enough space`, async ({
+    page,
+  }) => {
+    await page.goto(`/MiddleTextTruncation`);
+
+    const setContainerSize = getSetContainerSize(page);
+
+    const middleTextTruncation = page.getByTestId('middleTextTruncation');
+    await expect(middleTextTruncation.first()).toHaveText(longItem);
+
+    await setContainerSize('200px');
+
+    await expect(middleTextTruncation.first()).toHaveText(
+      'MyFileWithAReallyLon…2.html',
+    );
+
+    await setContainerSize(undefined);
+
+    // should restore hidden items when space is available again
+    expect(middleTextTruncation.first()).toHaveText(longItem);
```

Comment:
don't forget to `await`

Reply:
i noticed a few more `expect`s that were missing `await`s.

we might need a lint rule to help avoid this situation